BLHeliBootLoad.inc

```diff
@@ -1,385 +1,377 @@
-	anl	RTX_MDOUT, #NOT(1 SHL RTX_PIN)	; Disable pushpull
-	setb	RTX_PORT.RTX_PIN				; Set data high
-	mov	RTX_SKIP, #0FFh
-	mov	XBR2, #40h;					; Enable crossbar
-	; Set number of connect attempts before exiting bootloader
-	mov	Cmdh, #250
-
-	; Identifier scanning
-abd:	mov	Xl, #(low(BOOT_DELAY / 6)+1)
-	mov	Xh, #(high(BOOT_DELAY / 6)+1)
-	mov	Cmdl, #(high((BOOT_DELAY / 6) SHR 8)+1)
-	mov	Crcl, #0
-	mov	Crch, #0
-	mov	DPTR, #BOOT_SIGN
-	mov	Parah, #(BOOT_MSG - BOOT_SIGN)
-	mov	Baudl, #low(BAUDTIME)
-	mov	Baudh, #high(BAUDTIME)
+XTAL          EQU 25000000
+
+BOOT_START    EQU 1C00h                           ; Bootloader segment address
+BOOT_DELAY    EQU XTAL/4                          ; About 250ms (don't set to fast to avoid connection problems)
+BOOT_BAUDRATE EQU 19200                           ; Only used if no baudrate detection activated,XTAL is than important
+BOOT_VERSION  EQU 6                               ; Version 6 (must be not changed)
+BOOT_PAGES    EQU 1                               ; Number of flash segments for bootloader
+
+UART_LOOP     EQU 26                              ; Depends upon timing of putc,getc
+BAUDTIME      EQU ((XTAL/BOOT_BAUDRATE)/3)-UART_LOOP
+
+SUCCESS       EQU 030h
+ERRORVERIFY   EQU 0C0h
+ERRORCOMMAND  EQU 0C1h
+ERRORCRC      EQU 0C2h
+ERRORPROG     EQU 0C5h
+
+POLYNOM       EQU 0A001h                          ; CRC Polynom
+
+Xl            EQU R0                              ; Temporary X
+Xh            EQU R1
+Paral         EQU R2                              ; Params for UART
+Parah         EQU R3
+Cmdl          EQU R4                              ; Commands
+Cmdh          EQU R5
+Cntl          EQU R6                              ; Baudtime
+Cnth          EQU R7
+
+DSEG          AT  20h
+Bit_Reg:      DS  1                               ; Bit storage register
+Byte_Reg:     DS  1                               ; Byte storage register
+Crcl:         DS  1                               ; CRC 16Bit
+Crch:         DS  1
+Baudl:        DS  1                               ; Baudtime
+Baudh:        DS  1
+Bit_Cnt:      DS  1                               ; Counter in UART loops
+Byte_Cntl:    DS  1                               ; Generic counter
+Byte_Cnth:    DS  1
+BL_Flash_Key_1: DS 1                              ; Flash keys
+BL_Flash_Key_2: DS 1
+
+CSEG          AT  BOOT_START                      ; Bootloader start
+init:clr      IE_EA
+; Select register bank 0 for main program routines
+clr           PSW.3                               ; Select register bank 0 for main program routines
+; Disable the WDT.
+mov           WDTCN, #0DEh                        ; Disable watchdog
+mov           WDTCN, #0ADh
+; Initialize stack
+mov           SP, #0c0h                           ; Stack = 64 upper bytes of RAM
+; Initialize clock
+mov           CLKSEL, #00h                        ; Set clock divider to 1
+; Initialize VDD monitor
+orl           VDM0CN, #080h                       ; Enable the VDD monitor
```

Comment:
I am not sure if we should align the 2nd operand with all the second operand or if a ", " is good enough?

This would be the alternative, and the 2nd operand would have a global offset - similar as the first one has to the instruction:
```
orl           VDM0CN,    #080h                    ; Enable the VDD monitor
mov           Baudl,     #38h                     ; Wait 100us
mov           Baudh,     #03h
```

Reply:
I think that all that improves readability is good. Both formats seem to be good